expat/configure.ac

```diff
@@ -395,9 +395,18 @@ AC_SUBST([AM_CFLAGS])
 AC_SUBST([AM_CXXFLAGS])
 AC_SUBST([AM_LDFLAGS])
 
+dnl Force the use of CMAKE_SHARED_LIBRARY_PREFIX under CMake
+AC_MSG_CHECKING([for shared library prefix])
```

Comment:
It's not just shared libraries but also static ones. Is that correct?
Also "prefix" could use more distinguishing from `--prefix=<path>`.
How about checking "for library name prefix". That would address both.